quodlibet/util/collection.py

```diff
@@ -47,19 +48,73 @@
 ELPOEP = list(reversed(PEOPLE))
 PEOPLE_SCORE = [100 ** i for i in range(len(PEOPLE))]
 
+SMOOTHING_FACTOR_DEFAULT = 10
```

Comment:
This default seems like it's in two places now? `config.py` and here?

Reply:
True. I think I missed this, because when I wrote the default constant, I hadn't yet seen / remembered that there's even a default config.

Also the code already there also did something like:

```python
config.getfloat("settings", "bayesian_rating_factor", 0.0)
```

Despite there being the following in the default config:

```json
"bayesian_rating_factor": "0.0"
```

But you're right, so, I think it makes sense to remove them there too (default arguments of `getfloat("settings", "bayesian_rating_factor")`)